### DISCUSSION TITLE
Don't allow ide paths in published crates

### DISCUSSION THREAD

**Discussion by christopinka** [rust-lang/crates.io#6714] [Feature Requests]
Example is 'genevo' https://github.com/innoave/genevo. Has a .idea path and also in .gitignore.

Scan for these paths and ignores.

Main complaint is including non-project cruft goes against well known convention.

**Comment by Turbo87** [comment]
I guess this is something that cargo would implement. I'm not sure if we should do anything about it on the server side.
